### PR TITLE
DDF-1406 Fixes minor issues in deployment notes in README

### PR DESCRIPTION
**Pull request by roelens8**
Makes changes to the readme file of the DDF's How-to-build section.

@brendan-hofmann 
@coyotesqrl
@andrewkfiedler is the hero!
@pklinef 

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/110)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ set MAVEN_OPTS='-Xmx512m -XX:MaxPermSize=128m'
 
 
 ### How to build ###
-In order to run through a full build, be sure to have a clone for all repositories (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
+In order to run through a full build, be sure to have a clone for the ddf repository and optionally the ddf-support repository (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
 
 ```
 git clone git://github.com/codice/ddf.git
-git clone ddf-support  (https://github.com/codice/ddf-support.git)
+git clone https://github.com/codice/ddf-support.git (Optional)
```
Comment (coyotesqrl) [inline README.md]: Should change to the git: protocol to match ddf.

Comment (roelens8) [inline README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -56,13 +56,13 @@ set MAVEN_OPTS='-Xmx512m -XX:MaxPermSize=128m'
...
-In order to run through a full build, be sure to have a clone for all repositories (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
+In order to run through a full build, be sure to have a clone for the ddf repository and optionally the ddf-support repository (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
 
 ```
 git clone git://github.com/codice/ddf.git
-git clone ddf-support  (https://github.com/codice/ddf-support.git)
+git clone git://github.com/codice/ddf-support.git (Optional)
 ```
-Change to the top level directory of DDF source distribution.
+Change to the root directory of the newly installed ddf repository. Run the following command:
```
Comment (brendan-hofmann) [inline README.md]: Shouldn't this say 'cloned' instead of 'installed'?

**Comment by brendan-hofmann** [comment]
Could you fix the typos in the commit messages (or better, squash them)? The [commit guidelines](https://codice.atlassian.net/wiki/display/DDF/Git+Commit+Message+Guidelines) can help.

File: README.md (hunk 2 above)
Comment (roelens8) [inline README.md]: You're right it should.

**Comment by coyotesqrl** [comment]
I'm not sure there's much value in keeping multiple commits for this change; you could simply add more detail to the commit description (not the first line) regarding the code review changes and squash to a single commit as @brendan-hofmann suggests.

**Comment by roelens8** [comment]
Squashed them into one commit message, should be okay now.

**Comment by coyotesqrl** [comment]
:+1:

**Comment by brendan-hofmann** [comment]
Much nicer! :+1:

File: README.md [hunk 3]
```diff
@@ -56,13 +56,13 @@ set MAVEN_OPTS='-Xmx512m -XX:MaxPermSize=128m'
...
-In order to run through a full build, be sure to have a clone for all repositories (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
+In order to run through a full build, be sure to have a clone for the ddf repository and optionally the ddf-support repository (NOTE: daily snapshots are deployed so downloading and building each repo may not be necessary since those artifacts will be retrieved.):
 
 ```
 git clone git://github.com/codice/ddf.git
-git clone ddf-support  (https://github.com/codice/ddf-support.git)
+git clone git://github.com/codice/ddf-support.git (Optional)
 ```
-Change to the top level directory of DDF source distribution.
+Change to the root directory of the newly cloned ddf repository. Run the following command:
```
Comment (pklinef) [inline README.md]: Just my opinion but I think "the cloned ddf repository" would be clear enough without "newly".

**Comment by pklinef** [comment]
:+1:

File: README.md (hunk 3 above)
Comment (roelens8) [inline README.md]: Fixed.

**Pull request merged**